submissions/TaniaBondarenko/popup2_task/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hooly-style Popup</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <nav class="navigation">
+    <ul class="nav">
+         <li><button class="nav_icon"><img src="images/photos.png" class="nav_icon_image" alt="Photos icon"></button></li>
+         <li><button class="nav_icon"><img src="images/mail.png" class="nav_icon_image" alt="Mail icon"></button></li>
+         <li><button class="nav_icon"><img src="images/bell.png" class="nav_icon_image" alt="Bell icon"></button></li>
+    </ul>
+      <input id="checkbox" type="checkbox" class="hidden_checkbox">
+      <button class="nav_icon"><label for="checkbox"><img src="images/popup-button.png" class="nav_icon_image" alt="Menu button"></label></button>      
+<!--Popup content, which is shown/hidden after menu-button pressed-->
+    <ul class="main_menu_icons">
+          <li><a href="#"><img src="images/apps.png" class="li_icon_image" alt="Application icon"></a></li>
+          <li><a href="#"><img src="images/blogs.png" class="li_icon_image" alt="Blog icon"></a></li>
+          <li><a href="#"><img src="images/calendar.png" class="li_icon_image" alt="Calendar icon"></a></li>
+          <li><a href="#"><img src="images/chat.png" class="li_icon_image" alt="Chat icon"></a></li>
+          <li><a href="#"><img src="images/contacts.png" class="li_icon_image" alt="Contacts icon"></a></li>
+          <li><a href="#"><img src="images/maps.png" class="li_icon_image" alt="Maps icon"></a></li>
+          <li><a href="#"><img src="images/videos.png" class="li_icon_image" alt="Videos icon"></a></li>
+          <li><a href="#"><img src="images/tanslation.png" class="li_icon_image" alt="Translation icon"></a></li>
+          <li><a href="#"><img src="images/search.png" class="li_icon_image" alt="Search icon"></a></li>
+      <div class="show_more">
```

Comment:
Element `div` not allowed as child of element `ul` in this context. (Suppressing further errors from this subtree.)

Reply:
I'll check.